src/lib/tls/tls_algos.h

```diff
@@ -114,11 +121,21 @@ enum class Group_Params_Code : uint16_t {
    // https://datatracker.ietf.org/doc/draft-tls-westerbaan-xyber768d00/03/
    HYBRID_X25519_KYBER_512_R3_OQS = 0x2F39,
    HYBRID_X25519_KYBER_768_R3_OQS = 0x6399,
+   HYBRID_X25519_eFRODOKEM_640_SHAKE_OQS = 0x2F81,
+   HYBRID_X25519_eFRODOKEM_640_AES_OQS = 0x2F80,
 
    HYBRID_SECP256R1_KYBER_512_R3_OQS = 0x2F3A,
    HYBRID_SECP256R1_KYBER_768_R3_OQS = 0x639A,
+   HYBRID_SECP256R1_eFRODOKEM_640_SHAKE_OQS = 0x2F01,
+   HYBRID_SECP256R1_eFRODOKEM_640_AES_OQS = 0x2F00,
+
    HYBRID_SECP384R1_KYBER_768_R3_OQS = 0x2F3C,
+   HYBRID_SECP384R1_eFRODOKEM_976_SHAKE_OQS = 0x2F03,
+   HYBRID_SECP384R1_eFRODOKEM_976_AES_OQS = 0x2F02,
+
    HYBRID_SECP521R1_KYBER_1024_R3_OQS = 0x2F3D,
+   HYBRID_SECP521R1_eFRODOKEM_1344_SHAKE_OQS = 0x2F05,
+   HYBRID_SECP521R1_eFRODOKEM_1344_AES_OQS = 0x2F04,
```

Comment:
IMO it's a bit more readable if all the eFRODOKEM suites are grouped rather than sorting by NIST security level

Reply:
Done